pkgs/applications/misc/kaufkauflist/default.nix

```diff
@@ -0,0 +1,51 @@
+{ lib, buildGoModule, fetchFromGitHub, esbuild, buildNpmPackage, fetchFromGitea }:
+
+let
+  esbuild' = esbuild.override {
+    buildGoModule = args: buildGoModule (args // rec {
```

Comment:
```suggestion
  esbuild' = buildPackages.esbuild.override {
    buildGoModule = args: buildPackages.buildGoModule (args // rec {
```

Reply:
Why is that better than the current way?

Reply:
There is `buildPackages` before `buildNpmPackage` either.

Reply:
> Why is that better than the current way?

Currently, you can't cross-compile kaufkauflist, because the esbuild binary will be for the host system, when it needs to run on the build system.

Reply:
> buildNpmPackage doesn't have it because you're compiling for the host system.

Because kaufkauflist is compiled to web?

Reply:
There's no concept of a web target in Nixpkgs. Essentially, every package is expected to be different for every supported system (aarch64-linux, aarch64-darwin, etc.). Given this, when building `pkgsCross.aarch64-linux.kaufkauflist` on an x86_64-linux system without referencing `buildPackages`, the esbuild binary will be for aarch64-linux, which can't be run on the build system.